Adds rendering of a Thumbnail of an entry (#38)
echemdb/echemdb: doc/news/thumbnail.rst

**Added:**

> * Added `echemdb.cv.entry.Entry.thumbnail()` to render preview plots of the entry without axis.

Author: ```suggestion
* Added `echemdb.cv.entry.Entry.thumbnail()` which returns a thumbnail as bytes of the entry without axis.
```
